### ISSUE TITLE
Error occurs when emacs creates backup files

### ISSUE THREAD

**Issue by pcarbo** [workflowr/workflowr#47]
I have two files in my workflowr project repository, `analysis/temp.Rmd` and `analysis/temp.Rmd~`; the latter is [an emacs backup file](https://www.emacswiki.org/emacs/BackupDirectory). This is the error I get when trying to publish `temp.Rmd`:

```
> wflow_publish("temp.Rmd")
Error in relpath(path = p[i], start = start) :
  No tilde allowed in path or start
```

**Comment by jdblischak** [comment]
Good catch. I was trying to avoid absolute file paths that started with a tilde, but I was checking the entire filepath. I fixed this bug by only searching at the beginning of the filepaths. This bugfix is on the dev branch and will be included in the next release.

**Comment by pcarbo** [comment]
@jdblischak Great! Feel free to close the issue.

**Comment by jdblischak** [comment]
It'll close automatically once the commit is merged into the master branch (via a GitHub trick). It's useful to have it open until then so that other users that have the stable release installed know the bug is still present.

**Comment by jdblischak** [comment]
I screwed up the GitHub trick, but nevertheless this bug is fixed in [v0.7.0](https://github.com/jdblischak/workflowr/releases/tag/v0.7.0).
